### DISCUSSION TITLE
"Pause" Command

### DISCUSSION THREAD

**Discussion by ThomasSeeker** [smplayer-dev/smplayer#343] [Q&A]
Hey there, 

is there a way to send a "Pause"-Command to SMPlayer (via terminal or script) while it is playing in minimized mode? ... And also send it specifically to one instance only, even if other instances of SMPlayer are running (minimized or raised; playing or not).

Best greetings,
Thomas

**Comment by smplayer-dev** [comment]
`smplayer -send-action pause`

But this only works when using the one instance option.

**Reply by ThomasSeeker** [reply]
Thank you, Ricardo!

Too bad, that this doesn't work for multiple instances usage too ... 

But thank you anyway!

**Reply by smplayer-dev** [reply]
If you use linux, it's possible by using the mpris interface.

This pauses two instances:
```
$ qdbus | grep smplayer
 org.mpris.MediaPlayer2.smplayer
 org.mpris.MediaPlayer2.smplayer.instance5319

$ qdbus org.mpris.MediaPlayer2.smplayer /org/mpris/MediaPlayer2 Pause
$ qdbus org.mpris.MediaPlayer2.smplayer.instance5319 /org/mpris/MediaPlayer2 Pause

```

**Reply by ThomasSeeker** [reply]
Ohhhhh ... This is beautiful! ... Exactly what I was looking for!
Works like a charm ... :-)
Thank you very much!
